packages/types/src/Metadata/v4/latest.substrate.v4.json

```diff
@@ -142,6 +143,46 @@
               "documentation": [
                 " Events deposited for the current block."
               ]
+            },
+            {
+              "name": "EventCount",
+              "modifier": "Default",
+              "type": {
+                "PlainType": "EventIndex"
+              },
+              "fallback": "0x00000000",
+              "documentation": [
+                " The number of events in the `Events<T>` list."
+              ]
+            },
+            {
+              "name": "EventTopics",
+              "modifier": "Default",
+              "type": {
+                "DoubleMapType": {
+                  "hasher": "Blake2_256",
```

Comment:
Not convinced on the naming here of the hasher as opposed to line 167

Reply:
There's an issue on substrate to convert key2Hasher from Text to StorageHasher, should be fine once that's done.

Reply:
Ahh, cool.